app/players/dwarves/dwarf_blacksmith.py

```diff
@@ -0,0 +1,16 @@
+from app.players.dwarves.dwarf import Dwarf
+
+
+class DwarfBlacksmith(Dwarf):
+    def __init__(self, nickname: str,
+                 favourite_dish: str,
+                 skill_level: int) -> None:
+        super().__init__(nickname, favourite_dish)
+        self._skill_level = skill_level
+
+    def get_rating(self) -> int:
+        return self._skill_level
+
+    def player_info(self) -> str:
+        return (f"Dwarf blacksmith {self._nickname} "
+                f"with skill of the {self._skill_level} level")
```

Comment:
Use double quotes for consistency, as per the code style guideline: 'Use one style of quotes in your code. Double quotes are preferable.'